### ISSUE TITLE
Webpack builds fails with watchpack 1.3.0 [Fatal error: Cannot convert undefined or null to object]

### ISSUE THREAD

**Issue by readme42** [webpack/webpack#4341]
**Do you want to request a *feature* or report a *bug*?**
Bug report

**What is the current behavior?**
Webpack karma builds fail with:
```
22 02 2017 16:38:38.711:INFO [karma]: Karma v1.5.0 server started at http://0.0.0.0:8080/
22 02 2017 16:38:38.712:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
22 02 2017 16:38:38.799:INFO [launcher]: Starting browser PhantomJS
Fatal error: Cannot convert undefined or null to object
```

**If the current behavior is a bug, please provide the steps to reproduce.**
Reproducing here would take me hours to set up. Its an task running karma/jasmine tests written in typescript inside phantomjs.  I will do later if you tell me that you really need it.

**What is the expected behavior?**
Webpack builds should not fail

**Our investigation results in:** 

Webpack installed via npm yesterday: Webpack was installed with watchpack 1.2.1
Webpack installed via npm today: Webpack is installed with watchpack 1.3.0

Nailing down watchpack via `npm-shrinkwrap.json` "solves" our problem and all builds run fine again.

```
{
    "dependencies": {
        "webpack": {
            "version": "2.2.1",
            "dependencies": {
                "watchpack": {
                    "version": "1.2.1"
                }
            }
        }
    }
}

```

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**
Same on Linux and Windows.
Node 6.9.1
Webpack 2.2.1

**Comment by timse** [comment]
is there an easy way to reproduce this? (One that does not take hours :)) ?

**Comment by sokra** [comment]
I guess it's fixed in 1.3.1

**Comment by readme42** [comment]
Yeah it's fixed in 1.3.1, thanks a lot 👍
